### PR TITLE
clarify requirements for specs

### PR DESCRIPTION
**Pull request by michelleN**
resolves #219

### CODE REVIEW

**Comment by michelleN** [comment]
cc @lizrice

**Comment by caniszczyk** [comment]
RFC @cncf/toc - this is straightforward imho LGTM

**Comment by quinton-hoole** [comment]
LGTM

**Comment by xiang90** [comment]
lgtm

**Comment by michelleN** [comment]
@lizrice - do I need some amount of TOC approval before merging or is your lgtm sufficient given that we talked about it all in the meeting?

**Comment by yurishkuro** [comment]
Description says "resolves 219", but #219 is another PR, not an issue.

File: process/graduation_criteria.adoc [hunk 1]
```diff
@@ -18,6 +18,8 @@ To be accepted to incubating stage, a project must meet the sandbox stage requir
  * Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
  * Demonstrate a substantial ongoing flow of commits and merged contributions.
  * Since these metrics can vary significantly depending on the type, scope and size of a project, the TOC has final judgement over the level of activity that is adequate to meet these criteria
+ * A clear versioning scheme.
```
Comment (yurishkuro) [inline process/graduation_criteria.adoc]: "clear" is kind of subjective. What is the versioning scheme of `grep`? What is the ask here? Guideline? Links to examples? Is this looking for semver-style compatibility guarantees?

Comment (caniszczyk) [reply process/graduation_criteria.adoc]: I will ask the TOC for examples, but I'm assuming a documented release process and versioning scheme is all that's required

**Comment by caniszczyk** [comment]
There was no issues with this after being open for a few weeks, merging it in

**Pull request merged**